oracle/tests/test_e2e.py

```diff
@@ -0,0 +1,47 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+import pytest
+
+METRICS = [
+    'oracle.process.pga_maximum_memory',
+    'oracle.process.pga_used_memory',
+    'oracle.process.pga_freeable_memory',
+    'oracle.cursor_cachehit_ratio',
```

Comment:
The test runs fine, but got this assertion failure:

```
================================================================================ test session starts ================================================================================
platform darwin -- Python 2.7.16, pytest-4.6.4, py-1.8.0, pluggy-0.12.0 -- /Users/alexandre.yang/repos/dd/integrations-core/oracle/.tox/py27-oracle/bin/python
cachedir: .tox/py27-oracle/.pytest_cache
benchmark: 3.2.2 (defaults: timer=time.time disable_gc=False min_rounds=5 min_time=0.000005 max_time=1.0 calibration_precision=10 warmup=False warmup_iterations=100000)
rootdir: /Users/alexandre.yang/repos/dd/integrations-core/oracle
plugins: benchmark-3.2.2, cov-2.7.1, mock-1.10.4, datadog-checks-dev-0.31.1
collected 12 items / 11 deselected / 1 selected

tests/test_e2e.py::test_check FAILED                                                                                                                                          [100%]

===================================================================================== FAILURES ======================================================================================
____________________________________________________________________________________ test_check _____________________________________________________________________________________

dd_agent_check = <function run_check at 0x10c5a7410>

    @pytest.mark.e2e
    def test_check(dd_agent_check):
        aggregator = dd_agent_check()
        for metric in METRICS:
>           aggregator.assert_metric(metric)

tests/test_e2e.py:46:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
../datadog_checks_base/datadog_checks/base/stubs/aggregator.py:193: in assert_metric
    self._assert(condition, msg=msg, expected_stub=expected_metric, submitted_elements=self._metrics)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

condition = False, msg = "Needed at least 1 candidates for 'oracle.cursor_cachehit_ratio', got 0"
expected_stub = MetricStub(name='oracle.cursor_cachehit_ratio', type=None, value=None, tags=None, hostname=None)
submitted_elements = defaultdict(<type 'list'>, {u'oracle.tablespace.offline': [MetricStub(name=u'o...:oracle@oracle-alexandre-yang-rh1o0xhw (P004)'], hostname=u'docker-desktop')]})

    @staticmethod
    def _assert(condition, msg, expected_stub, submitted_elements):
        new_msg = msg
        if not condition:  # It's costly to build the message with similar metrics, so it's built only on failure.
            new_msg = "{}\n{}".format(msg, build_similar_elements_msg(expected_stub, submitted_elements))
>       assert condition, new_msg
E       AssertionError: Needed at least 1 candidates for 'oracle.cursor_cachehit_ratio', got 0
E         Expected:
E                 MetricStub(name='oracle.cursor_cachehit_ratio', type=None, value=None, tags=None, hostname=None)
E         Similar submitted:
E         Score   Most similar
E         0.86    MetricStub(name=u'oracle.buffer_cachehit_ratio', type=0, value=99.47377014160156, tags=[], hostname=u'docker-desktop')
E         0.84    MetricStub(name=u'oracle.library_cachehit_ratio', type=0, value=84.85789489746094, tags=[], hostname=u'docker-desktop')
E         0.68    MetricStub(name=u'oracle.memory_sorts_ratio', type=0, value=100, tags=[], hostname=u'docker-desktop')
E         0.53    MetricStub(name=u'oracle.physical_reads', type=0, value=387.40399169921875, tags=[], hostname=u'docker-desktop')
E         0.50    MetricStub(name=u'oracle.tablespace.in_use', type=0, value=3.7200944423675537, tags=[u'tablespace:EXAMPLE'], hostname=u'docker-desktop')
E         0.50    MetricStub(name=u'oracle.tablespace.in_use', type=0, value=2.3855221271514893, tags=[u'tablespace:SYSTEM'], hostname=u'docker-desktop')
E         0.50    MetricStub(name=u'oracle.tablespace.in_use', type=0, value=1.7452248334884644, tags=[u'tablespace:SYSAUX'], hostname=u'docker-desktop')
E         0.50    MetricStub(name=u'oracle.tablespace.in_use', type=0, value=0.23498547077178955, tags=[u'tablespace:UNDOTBS1'], hostname=u'docker-desktop')
E         0.50    MetricStub(name=u'oracle.tablespace.in_use', type=0, value=0.0051498436369001865, tags=[u'tablespace:USERS'], hostname=u'docker-desktop')
E         0.50    MetricStub(name=u'oracle.tablespace.in_use', type=0, value=0, tags=[u'tablespace:TEMP'], hostname=u'docker-desktop')
E         0.48    MetricStub(name=u'oracle.temp_space_used', type=0, value=0, tags=[], hostname=u'docker-desktop')
E         0.48    MetricStub(name=u'oracle.tablespace.size', type=0, value=34359721984, tags=[u'tablespace:USERS'], hostname=u'docker-desktop')
E         0.48    MetricStub(name=u'oracle.tablespace.size', type=0, value=34359721984, tags=[u'tablespace:UNDOTBS1'], hostname=u'docker-desktop')
E         0.48    MetricStub(name=u'oracle.tablespace.size', type=0, value=34359721984, tags=[u'tablespace:TEMP'], hostname=u'docker-desktop')
E         0.48    MetricStub(name=u'oracle.tablespace.size', type=0, value=34359721984, tags=[u'tablespace:SYSTEM'], hostname=u'docker-desktop')
E       assert False

../datadog_checks_base/datadog_checks/base/stubs/aggregator.py:235: AssertionError
====================================================================== 1 failed, 11 deselected in 3.37 seconds ======================================================================
ERROR: InvocationError for command /Users/alexandre.yang/repos/dd/integrations-core/oracle/.tox/py27-oracle/bin/pytest -v (exited with code 1)
______________________________________________________________________________________ summary ______________________________________________________________________________________
ERROR:   py27-oracle: commands failed

Failed!
```

Reply:
Yeah apparently some are not available right away. Removed them for now.